convention-develocity-shared/convention-develocity-shared/src/main/java/com/myorg/DevelocityConventions.java

```diff
@@ -0,0 +1,33 @@
+package com.myorg;
+
+import com.myorg.configurable.BuildCacheConfigurable;
+import com.myorg.configurable.BuildScanConfigurable;
+import com.myorg.configurable.DevelocityConfigurable;
+
+final class DevelocityConventions {
+
+    void configureDevelocity(DevelocityConfigurable develocity) {
+        // CHANGE ME: Apply your Develocity configuration here
+        develocity.setServer("https://develocity-samples.gradle.com");
```

Comment:
Not sure if it would make sense to offer a way to use this directly without an obligation to change the code / recompile / redeploy (although it is not too involved).

I am guessing a lib which would contain a configuration file and ship the convention-develocity which would load data from this configuration file if present could be published instead.

Reply:
My point is that this reference could be even more practical if you wouldn't require to fork/change the Java code to use it.
A customer could create its DV custom extension as a lib containing a configuration file and depending on the convention-develocity lib. 
This was just a thought though.

Reply:
Ah, I understand what you mean now.

It's an interesting thought, but whether it's updating a properties file, using a library, or copying/pasting the contents of this sample, they must create a new repository to tailor it to their needs. The latter gives complete control over the behavior so I think we should stick with that.

Where I think we can improve is to externalize the "boilerplate" adapter code. As it is right now, it creates a lot of noise in the sample.